cli/src/test/java/com/devonfw/tools/ide/version/VersionRangeTest.java

```diff
@@ -109,9 +109,12 @@ public void testContains() {
     checkContainsNot("(11,22)", "10*");
     checkContains("(11,22)", "11*");
     checkContains("(11,22)", "11.0*");
-    checkContains("(11,22)", "22*");
-    checkContains("(11,22)", "22.*");
+    checkContains("(11,22)", "21.99*");
+    checkContainsNot("(11,22)", "22*");
+    checkContainsNot("(11,22)", "22.*");
     checkContainsNot("(11,22)", "23*");
+    checkContains("(11,22)", "21.99*");
```

Comment:
```suggestion
    checkContains("(11,22)", "21.99.*");
```
Due this line is identical to line 122 I think this is a copy/paste error and you forgot to add a "<dot>" between "99" and "*".